applications/precompiled_pva/cpp/main.cpp

```diff
@@ -0,0 +1,156 @@
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "gxf/std/tensor.hpp"
+#include "holoscan/holoscan.hpp"
+#include "pva_unsharp_mask/pva_unsharp_mask.hpp"
+
+#include <holoscan/operators/holoviz/holoviz.hpp>
+#include <holoscan/operators/video_stream_recorder/video_stream_recorder.hpp>
+#include <holoscan/operators/video_stream_replayer/video_stream_replayer.hpp>
+#include <holoscan/core/system/gpu_resource_monitor.hpp>
+
+#include <iostream>
+#include <string>
+
+namespace holoscan::ops {
+class PreCompiledPVAExecutor : public Operator
+{
+public:
+    HOLOSCAN_OPERATOR_FORWARD_ARGS(PreCompiledPVAExecutor);
+    PreCompiledPVAExecutor() = default;
+
+    void setup(OperatorSpec &spec) override
+    {
+      //   HOLOSCAN_LOG_INFO("PreCompiledPVAExecutor::setup");
```

Comment:
Remove commented-out lines or keep in as debug logging?

Reply:
Removed the line